eservice-versions-archiver/src/main/scala/it/pagopa/interop/eserviceversionsarchiver/ApplicationConfiguration.scala

```diff
@@ -0,0 +1,36 @@
+package it.pagopa.interop.eserviceversionsarchiver
+
+import com.typesafe.config.{Config, ConfigFactory}
+import it.pagopa.interop.commons.cqrs.model.ReadModelConfig
+import it.pagopa.interop.commons.utils.CORRELATION_ID_HEADER
+
+import java.util.UUID
+import java.util.concurrent.{ExecutorService, Executors}
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor}
+
+object ApplicationConfiguration {
+
+  implicit val context: List[(String, String)]    = (CORRELATION_ID_HEADER -> UUID.randomUUID().toString) :: Nil
+  private val blockingThreadPool: ExecutorService =
+    Executors.newFixedThreadPool(Runtime.getRuntime.availableProcessors())
+  val blockingEC: ExecutionContextExecutor        = ExecutionContext.fromExecutor(blockingThreadPool)
+  implicit val ec: ExecutionContext               = blockingEC
```

Comment:
this class is intended to be strictly related to the load of external configurations.
I suggest to move the above definitions elsewhere (generally the `Main` file)

Reply:
> this class is intended to be strictly related to the load of external configurations. I suggest to move the above definitions elsewhere (generally the `Main` file)

If I move those it gives me the following error:  "Selecting value ec (same for `blockingEC` and `context`) from object Main, which extends scala.DelayedInit, is likely to yield an uninitialized value"